index.html

```diff
@@ -2389,6 +2389,20 @@
 
 <section class="appendix" id="bp-summary"></section>
 
+<section class="appendix">
+
+# Conformance
+
+This document does not adhere to strict [[?RFC2129]] terminology because it is primarily of
+an informative nature and does not easily lend itself to constraining a conformance class.
+However, within the formulation of its principles, we have taken care to use "should" to indicate 
+that a principle can be ignored in some rare cases given that there are valid reasons for doing so and 
```

Comment:
I don't think we mean "can be ignored", but rather that a principle could be overridden in particular cases where there are valid reasons for doing so